src/MDagPath.inl

```diff
@@ -4,70 +4,48 @@ py::class_<MDagPath>(m, "DagPath")
     .def(py::self == MDagPath())
 
     .def("apiType", [](MDagPath & self) -> MFn::Type {
-        MStatus status = MStatus();
-        MFn::Type type = self.apiType(&status);
-
-        if (!status){
```

Comment:
I think we should still test it, just in case there is some edge case whereby it *does* actually fail. We could test both, would be no harm in that.